pymc/tests/logprob/utils.py

```diff
@@ -34,14 +34,52 @@
 #   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 #   SOFTWARE.
 
+from typing import Optional
+
 import numpy as np
 
-from pytensor import tensor as at
+from pytensor import tensor as pt
 from pytensor.graph.basic import walk
 from pytensor.graph.op import HasInnerGraph
+from pytensor.tensor.var import TensorVariable
 from scipy import stats as stats
 
-from pymc.logprob.abstract import MeasurableVariable, icdf, logcdf, logprob
+from pymc.logprob import factorized_joint_logprob
+from pymc.logprob.abstract import (
+    MeasurableVariable,
+    get_measurable_outputs,
+    icdf,
+    logcdf,
+    logprob,
+)
+from pymc.logprob.utils import ignore_logprob
+
+
+def joint_logprob(*args, sum: bool = True, **kwargs) -> Optional[TensorVariable]:
```

Comment:
Can we get rid of this one? How much test refactoring does it require?

Reply:
Quite a lot I think, but still doable. I don't think that any other logp function implements this sum thing, so I guess I could add this `pt.sum` manually to every test function that requires it. Let me know if you think that this is the right call, because it is a bit of work :) (which I would be happy to do if needed)

Reply:
Let's leave it then :)